table/block_based/block_based_table_builder.cc

```diff
@@ -626,6 +626,14 @@ struct BlockBasedTableBuilder::Rep {
     } else {
       base_context_checksum = 0;
     }
+
+    if (alignment > 0 && compression_type != kNoCompression) {
```

Comment:
is there a meaningful difference between using table_options.block_align and alignment here?

Reply:
I don't think so. I think they'll either both be zero or both be nonzero.